datasets/br/bcb_disqualified_persons/bcb_disqualified_persons.yaml

```
@@ -0,0 +1,27 @@
+title: Brazil's Central Bank General Register of Disqualified Persons  
+entry_point: crawler.py
+prefix: bcb-disqualified-persons
```

Since this prefixes each key, we try to keep it short but unique. We also prefix with the 2-letter country code where it's country-specific.

```suggestion
prefix: br-qgi
```

---

QGI is the acronym for Quadro Geral de Inabilitados (which is General Register of Disqualified Persons). In those cases, we use the local language acronym?